### PR TITLE
Improved README

### PR DESCRIPTION
**Pull request by olbapbetan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,11 +92,11 @@ yarn
 cd ios && pod install
 ```
 
-4. It is required to have `WebRTC.framework` M112 builds. You can find it in `libWebRTC.zip` inside the `ios` folder.
+1. It is required to have `WebRTC.framework` M112 builds. You can find it in `ios/libWebRTC.zip`.
```
Comment (Santiago-Souto) [inline README.md:95]: This should still be 4.

File: README.md [hunk 2]
```diff
@@ -85,11 +92,11 @@ yarn
...
 
-5. To download the zip file from Git LFS is needed to run the command `git lfs pull`.
+2. To download the zip file from Git LFS is needed to run the command `git lfs pull`.
```
Comment (Santiago-Souto) [inline README.md:97]: Same as above, should keep been 5.

File: README.md [hunk 3]
```diff
@@ -85,11 +92,11 @@ yarn
...
 
-3. Unzip libWebRTC inside the `ios` folder from this project.
+3. Unzip `libWebRTC` inside the `ios` folder from this project.
```
Comment (Santiago-Souto) [inline README.md:99]: In this case, 3. should be replace with a 6.

Comment (Santiago-Souto) [reply README.md:99]: In line 107, 1. should be replaced with a 5.

File: README.md [hunk 4]
```diff
@@ -128,6 +135,8 @@ Also, add the framework in `Build Phases -> Embed Frameworks` and on `Link Binar
 
 ![Adding WebRTC local reference in Pods target](assets/WebRTC-References-Pods.png)
 
+###### For Simulator only (*):
+
 - In `Frameworks Search Paths`, insert the following line: 
 ```
 $(PROJECT_DIR)/../libWebRTC/WebRTC.xcframework/tvos-arm64_x86_64-simulator
```
Comment (Santiago-Souto) [inline README.md:142]: Here it should be `tvos-x64-simulator` in order to include both simulator archs. Same in the `Header Search Paths`replacing `tvos-arm64` with `tvos-x64-simulator`

File: README.md [hunk 5]
```diff
@@ -140,11 +149,28 @@ $(PROJECT_DIR)/../libWebRTC/WebRTC.xcframework/tvos-arm64/WebRTC.framework/Heade
...
 6. Select `TestApp-tvOS` project and use a tvOS simulator with tvOS 16.
 
 7. Run the project, you should see the simulator with the app home page with a buttom to subscribe to a stream.
 
-To navigate use the arrow keys and enter button. Also, on the Simulator window you can go to Window -> Show Apple TV Remote and use it.
+To navigate use the arrow keys and enter button. Also, on the Simulator window you can go to `Window -> Show Apple TV Remote` and use it.
+
+#### Remarks
+(*) The frameworks and header search paths are exclusive. If a developer is working on a platform, this developer needs to remove the search paths for the other platform.
+
```
Comment (Santiago-Souto) [inline README.md:173]: Should we add as a remark the different architecture for each Mac machine?

**Review by Santiago-Souto — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -113,7 +120,7 @@ $(PROJECT_DIR)/../libWebRTC/WebRTC.xcframework/ios-arm64_x86_64-simulator/WebRTC
```
Comment (montanaeli) [inline README.md:120]: We have x64 builds also, we should consider them at the moment of adding the Search Paths

File: README.md [hunk 7]
```diff
@@ -85,11 +92,11 @@ yarn
...
+1. It is required to have `WebRTC.framework` M112 builds. You can find it in `ios/libWebRTC.zip`.
 
-5. To download the zip file from Git LFS is needed to run the command `git lfs pull`.
+2. To download the zip file from Git LFS is needed to run the command `git lfs pull`.
 
-3. Unzip libWebRTC inside the `ios` folder from this project.
+3. Unzip `libWebRTC` inside the `ios` folder from this project.
 #### iOS
 
 1. Open Xcode.
```
Comment (montanaeli) [inline README.md:102]: It should be mentioned for adding WebRTC as a framework step to check which Mac architecture is being used.

File: README.md (hunk 1 above)
Comment (olbapbetan) [reply README.md:95]: You're right. I don't know why that changed.

File: README.md (hunk 7 above)
Comment (olbapbetan) [reply README.md:102]: Good idea. I added a link to the remark (**).

File: README.md [hunk 8]
```diff
@@ -128,23 +139,44 @@ Also, add the framework in `Build Phases -> Embed Frameworks` and on `Link Binar
...
 ```
-$(PROJECT_DIR)/../libWebRTC/WebRTC.xcframework/tvos-arm64_x86_64-simulator
+$(PROJECT_DIR)/../libWebRTC/WebRTC.xcframework/tvos-x64-simulator
 $(PROJECT_DIR)/../libWebRTC/WebRTC.xcframework/tvos-arm64-simulator
 ```
 
 - In `Header Search Paths`, insert the following lines: 
 ```
-$(PROJECT_DIR)/../libWebRTC/WebRTC.xcframework/tvos-arm64/WebRTC.framework/Headers
+$(PROJECT_DIR)/../libWebRTC/WebRTC.xcframework/tvos-x64-simulator/WebRTC.framework/Headers
```
Comment (montanaeli) [inline README.md:152]: Ditto.

**Review by Santiago-Souto — APPROVED** [review]
LGTM

**Pull request merged**